SPECS/rubygem-openssl/rubygem-openssl.spec

```diff
@@ -13,6 +13,8 @@ URL:            https://github.com/ruby/openssl
 Source0:        https://github.com/ruby/openssl/archive/refs/tags/v%{version}.tar.gz#/%{gem_name}-%{version}.tar.gz
 BuildRequires:  git
 BuildRequires:  ruby
+Provides:       openssl = %{version}-%{release}
```

Comment:
The provides for the base gem names look incorrect, especially on this one which would override our `openssl` package. Looking at the state of the, should these be `rubygem-%{gem_name}`?